### PR TITLE
Iv 4970 time sync

### PR DESCRIPTION
**Pull request by psschroeter**
Add in ntp sync script for RL10. By default this script only tries to only setup NTP is an NTP config is missing. On Google and Ec2 clouds at least, NTP is generally already setup via DHCP or agent to go to the 169.254 servers which is local and more reliable across more situations (VPC etc) so try and respect that.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 Changelog for the RightLink10 Base ServerTemplate
 =================================================
 
-10.5.4
+unreleased changes
 ------
 - Moved UPGRADES_LOCATION parameter from rll/upgrade.sh to rll/setup-automatic-upgrade.sh and from rlw/upgrade.sh to rlw/setup-automatic-upgrade.sh. Added a new parameter UPGRADE_VERSION to rll/upgrade.sh and rlw/upgrade.sh to accept a simple text version to upgrade to, i.e. '10.5.4'
+- Add setup-ntp.sh and setup-ntp.ps1 to Boot scripts. These scripts setup the NTP daemon to synchonize system time, by default to time.rightscale.com. Accurate time is important for monitoring and proper functioning of the client.
```
Comment (lopaka) [inline CHANGES.md]: typo: synchronize

File: rll/enable-monitoring.sh [hunk 2]
```diff
@@ -272,6 +272,18 @@ else
     retry_command sudo apt-get update -y
     retry_command sudo apt-get install -y curl collectd-core
   elif [[ -d /etc/yum.repos.d ]]; then
+    # Workaround for broken collectd
```
Comment (lopaka) [inline rll/enable-monitoring.sh:275]: Do we want to add this change to the `CHANGES.md` file?

**Review by lopaka — CHANGES_REQUESTED** [review]

File: rll/enable-monitoring.sh (hunk 2 above)
Comment (psschroeter) [reply rll/enable-monitoring.sh:275]: yup, added

Comment (lopaka) [reply rll/enable-monitoring.sh:275]: :+1:

**Review by lopaka — APPROVED** [review]

**Comment by lopaka** [comment]
LGTM

**Pull request merged**